ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -51,6 +55,12 @@ chain:
       AWS_GUEST_INFRA_CREDENTIALS_FILE="/etc/hypershift-ci-jobs-awscreds/credentials"
       DEFAULT_BASE_DOMAIN=ci.hypershift.devcluster.openshift.com
       
+      HC_REGION=${HYPERSHIFT_AWS_REGION:-$LEASED_RESOURCE}
```

Comment:
What sets the LEASED_RESOURCE variable?

Reply:
@csrwng Thanks for the review. 

Quoting from the [docs:](https://docs.ci.openshift.org/docs/architecture/quota-and-leases/)
Whenever ci-operator runs a test target that has a cluster_profile set, a lease will be acquired before the test steps are executed. ci-operator will acquire the lease, present the name of the leased resource to the job in the ${LEASED_RESOURCE} environment variable ...

For example, when a job that uses the "aws-qe" cluster profile is executed, the CI operator acquires a lease from the "aws-qe-quota-slice", as detailed [here](https://github.com/openshift/release/blob/4e0eaed0b4c0ed86cef851dd0629d6312ca640a9/core-services/prow/02_config/_boskos.yaml#L710-L742). For instance, the lease might be "us-east-1--aws-qe-quota-slice-00". In this scenario, the steps within the job will have $LEASED_RESOURCE set to the part of the lease string before the "--", which would be "us-east-1".

In the case of AWS, $LEASED_RESOURCE denotes the region where a job puts cloud resources (e.g. the management cluster) in.